Google.PowerShell/Compute/GceManagedInstanceGroupCmdlets.cs

```diff
@@ -408,25 +434,90 @@ protected override void ProcessRecord()
                     break;
                 case ParameterSetNames.ByObject:
                     manager = Object;
+                    // Uses Zone by default unless user explicitly uses -Region.
+                    isZoneInstanceGroup = !MyInvocation.BoundParameters.ContainsKey(nameof(Region));
```

Comment:
Should this be `!string.isNullOrEmpty(Zone) && !MyInvocation.BoundParameters.ContainsKey(nameof(Region));`?
Should it also be after the `ProcessPipedObjectArguments()` call?